pkg/metrics/metrics.go

```diff
@@ -0,0 +1,63 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+const (
+	separator = "_"
+
+	subsystem = "stonesoup_sprayproxy"
+
+	github                    = "github"
+	githubRequestsName        = subsystem + separator + github + separator + "requests_total"
+	forwarded                 = "forwarded"
+	forwardedRequestsName     = subsystem + separator + forwarded + separator + "requests_total"
+	responseTime              = "response" + separator + "time"
+	forwardedResponseTimeName = subsystem + separator + responseTime + separator + "duration_seconds"
+
+	MetricsPort = 6000
+)
+
+var (
+	githubRequestCounter, forwardedRequestCounter = createCounters()
+	responseTimeHistogram                         = createHistogram()
+)
+
+func init() {
+	prometheus.MustRegister(githubRequestCounter)
+	prometheus.MustRegister(forwardedRequestCounter)
+	prometheus.MustRegister(responseTimeHistogram)
+}
+
+func createCounters() (prometheus.Counter, prometheus.Counter) {
```

Comment:
Rename to `createMetrics`, return an array that satisfies the interface for `MustRegister`

```suggestion
func createMetrics() ([]prometheus.Collector) {
```

Reply:
ok, but I will tweak you suggestion slightly (most likely was your intent anyway) and include the createHistogram in that as well